test/integration/targets/connection_posix/test.sh

```diff
@@ -16,3 +16,14 @@ INVENTORY="../connection_${group}/test_connection.inventory" ./test.sh \
     -e local_tmp=/tmp/ansible-local \
     -e remote_tmp=/tmp/ansible-remote \
     "$@"
+
```

Comment:
A posteriori, I think this addition should be moved in `connection_docker/test.sh`, using something like that:
```
#!/bin/bash
../connection_posix/test.sh
DOCKER_VERSION=$(docker version -f '{{ .Client.Version }}')
CWD=/var/tmp
[...]
```

Reply:
I agree. I apparently didn't look closely enough, I thought it already was there.